src/scss/hpe/_hpe.defaults.scss

```diff
@@ -16,7 +16,7 @@ $brand-status-colors: (
 );
 $brand-link-color: inherit;
 $link-hover-color: #000;
-$brand-link-text-decoration: underline;
+$brand-link-text-decoration: none;
```

Comment:
The general brand guideline is to have underlined anchors. So, I think we need to find another solution.